tests/test_api.py

```diff
@@ -981,14 +985,140 @@ def test_empty_to_uuid(self):
             api.move_identity('03877f31261a6d1a1b3971d240e628259364b8ac', '')
 
 
+class TestAddOrganization(TestCase):
+    """Unit tests for add_organization"""
+
+    def test_add_new_organization(self):
+        """Check if everything goes OK when adding a new organization"""
+
+        organization = api.add_organization(name='Example')
+
+        # Tests
+
+        self.assertIsInstance(organization, Organization)
+
```

Comment:
Remove this black line too. Please group similar statements without using blank lines between them. It's clear for reading.

Reply:
My bad, I wasn't sure if this was related with the next check. 
Done.